README.md

```diff
@@ -172,7 +172,7 @@ function Home() {
 Now, in our `Home` component, we can use the `useEventListener` hook to listen for our `user-entered` and `user-exited` events we're sending from our session backend.
 
 ```ts filename="src/components/Home.tsx"
-import { SessionBackendProvider, useEventListener } from '@jamsocket/javascript/react'
+import { useEventListener } from '@jamsocket/javascript/socketio'
```

Comment:
I think we should show this in context with the `import { SessionBackendProvider } from '@jamsocket/javascript/react'` right above it. When we do that, we should also highlight just the new line.